### DISCUSSION TITLE
VICTIM OF Fake GitHub Job Opportunity Email #109171

### DISCUSSION THREAD

**Discussion by crrusucosmin** [community/community#109561] [Other Feature Feedback, Questions, & Ideas] [answered]
### Body

Earlier this week, there has been an attack on GitHub that basically sent spam claiming that there's a job opportunity selected for the engineers that contributed to certain GitHub repositories. At first glance, everything looked safe, so I fall into the trap.

Later, they basically stole my GitHub access tokens (identity) and posted the same spammy message on other GitHub repositories. Later that day, my account has been disabled and now I cannot work. I have a lot of repositories that I contributed to - both private and open source.

In addition, I'm part of a few organizations and I cannot commit to their repositories either. I opened a ticket to reinstantiate my account (rusucosmin) but I still got no answer. Any idea how long this will take?

Fraud Alert: Fake GitHub Job Opportunity Email #109171

Weirdly enough, the initial GitHub thread explaining the issue no long works:

 https://github.com/orgs/community/discussions/109171

GitHub
https://github.com › orgs › community › discussions
3 days ago — It's a scam email from an impersonator of GitHub, claiming you have been selected to work for GitHub. You can easily notice that the domain of ...

**Comment by elGuille** [comment]
https://github.com/orgs/community/discussions/109171 was a post WARNING EVERYBODY about the fraud!!!
Now the account of the person warning us is suspended instead of the actual scammer?? 

How can we tag support?

**Comment by ettaboyle** [comment, marked as answer]
We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:
- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.  
- GitHub’s recruitment process would never mention its users via issues/PRs or other public content.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps). 
This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.

**Comment by KizitoK** [comment]
![image](https://github.com/community/community/assets/161340169/ad0eb5e5-2e35-4b80-819c-8e6cdf035eb4)

I tried warning someone about this and my account Kizito007 got suspended 😭 please I need assistance for recovery. It's almost a week

Support Ticket: #2622760

**Reply by ettaboyle** [reply]
Hi @KizitoK - The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). We are not able to help with account specific issues in our public community discussions.

**Comment by KizitoK** [comment]
Hello Amanda,

Please help me. I've opened the ticket since last week but still no
response.

Here's the ticket number: 2622760

Thanks
